### ISSUE TITLE
How would this be used to ((link or merge)) more than one account together in Asp.NET Core?

### ISSUE THREAD

**Issue by ExperimentsAndIdeas** [scottbrady91/AspNetCore-SignInWithApple-Example#5]
As part of a migration strategy, consolidation, or other need, what would be needed to allow:

Scenario 1: User may link multiple accounts together (Azure Active Directory, or B2C) while Apple ID or any other Issuer or provider, and then use the resource token in the background to access Outlook.office.com data via the Graph API?

Scenario 2: A user has a MSAL 2.0 token and would like Apple ID token in order to access data stored within CloudKit key-value store?

TL;DR I'm looking for ways to use OIDC background tokens with the other provider (log in with Apple, then access 1.0 Beta Graph API AND Cloudkit)
